Standardize BARTpho model card: badges, new examples, fixed broken im…

eshwanthkartitr opened the pull request:
Updated bartpho.md

# What does this PR do?

Fixes # (issue)
Standardize BARTpho model card


## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?


## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.


Please review this @stevhliu

eshwanthkartitr committed:
Standardize BARTpho model card: badges, new examples, fixed broken image section, and links (#36979)Update bartpho.md

eshwanthkartitr commented:
Hi @stevhliu,

Thank you for your dedication to maintaining high-quality documentation across the repository. I've standardized the BARTpho model card with several key improvements:

**Key Changes:**
- Standardized model card badges for consistency with other model documentation
- Added new practical examples to help users better understand implementation
- Fixed broken links and images that were impacting user experience

I'd appreciate your feedback on whether any additional elements would enhance the documentation further - specifically, would an architecture diagram improve clarity for users? I'm also open to any other suggestions you might have to make this model card even more valuable for the community.

Looking forward to your review and any insights you can share!

Best regards

stevhliu reviewed (commented):
Good start! Next time, make sure the code examples run and are aligned

eshwanthkartitr committed:
Update bartpho.md

Removed non-required/unsupported sections: Quantization, Attention visualizer, and Resources (plus stray tokenizer header).

Added code snippets which were suggested

stevhliu commented on docs/source/en/model_doc/bartpho.md:
```diff
@@ -16,78 +16,124 @@ rendered properly in your Markdown viewer.
+<img alt="Hugging Face Model" src="https://img.shields.io/badge/Model%20Hub-BARTpho-blue">
+<img alt="License" src="https://img.shields.io/badge/License-Apache%202.0-green">
+<img alt="Language" src="https://img.shields.io/badge/Language-Vietnamese-orange">
+</div>
```
You only need to remove the ones I highlighted. The other labels (PyTorch, TensorFlow, Flax) are still valid

eshwanthkartitr committed:
Update bartpho.md

Updated with necessary tags

stevhliu committed:
Update bartpho.md

HuggingFaceDocBuilderDev commented:
The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/transformers/pr_40051). All of your documentation changes will be reflected on that endpoint. The docs are available until 30 days after the last update.

stevhliu committed:
Update bartpho.md

stevhliu reviewed (approved):
Thanks!
